docs/additional-functionality/qualification-profiling-tools.md

```diff
@@ -0,0 +1,708 @@
+---
+layout: page
+title: Qualification and Profiling tools
+parent: Additional Functionality
+nav_order: 7
+---
+# Spark Qualification and Profiling tools
+
+The qualification tool is used to look at a set of applications to determine if the RAPIDS Accelerator for Apache Spark 
+might be a good fit for those applications.
+
+The profiling tool generates information which can be used for debugging and profiling applications.
+Information such as Spark version, executor information, properties and so on. This runs on either CPU or
+GPU generated event logs.
+
+(The code is based on Apache Spark 3.1.1 source code, and tested using Spark 3.0.x and 3.1.1 event logs)
```

Comment:
lets just remove this line, if anything we can move the based on spark 3.1.1 source code to below but really user just cares what spark they need and what event logs versions it supports

Reply:
Yes just removed that line.